Library/Formula/docker-machine-driver-xhyve.rb

```diff
@@ -0,0 +1,37 @@
+require "language/go"
+
+class DockerMachineDriverXhyve < Formula
+  desc "Docker Machine driver for xhyve"
+  homepage "https://github.com/zchee/docker-machine-driver-xhyve"
+  url "https://github.com/zchee/docker-machine-driver-xhyve/archive/v0.2.1.tar.gz"
+  sha256 "11ed10c4ea45249e06c9efad0abf1c91b0ec9c4c2a0818ade78f19f41a17794c"
+
+  head do
+    url "https://github.com/zchee/docker-machine-driver-xhyve.git"
+  end
+
+  depends_on :macos => :yosemite
+  depends_on "go" => :build
+  depends_on "docker-machine"
+
+  def install
+    contents = Dir["{*,.git,.gitignore}"]
+    gopath = buildpath/"gopath"
+    (gopath/"src/github.com/zchee/docker-machine-driver-xhyve").install contents
+
+    ENV["GOPATH"] = gopath
+    ENV["GO15VENDOREXPERIMENT"] = "1"
+
+    cd gopath/"src/github.com/zchee/docker-machine-driver-xhyve" do
+      if build.head?
+        ENV["GIT_COMMIT"] = `git rev-parse --short HEAD 2>/dev/null`.delete!("\n")
```

Comment:
It’s probably better to use a local variable rather than the environment if this is not needed outside of Ruby.